psi4/driver/p4util/util.py

```diff
@@ -33,8 +33,11 @@
 import warnings
 
 from psi4 import core
+from psi4.driver.procrouting import *
 from .exceptions import ValidationError
 
+def tdscf(wfn,**kwargs):
+    return proc.run_tdscf_energy(ref_wfn = wfn)        
```

Comment:
calling a high-level fn (from `proc`) from a menial file (like `util`) seems like risking a circular import now or in the future. I'd have no problem with a two-liner like this is driver.py or proc.py, wherever more convenient.